### PR TITLE
Organization signup not ready page closes #4246

### LINKED ISSUE #4246: Error on signup if organization seats are reached
If a user tries to signup in an organization without remaining seats, redirect to an error page saying "Please, contact with the admin of your organization (admin_email) to begin your signup process."

### PR DESCRIPTION
**Pull request by juanignaciosl**
@xavijam CR this, please. This closes #4246.

### CODE REVIEW

**Comment by xavijam** [comment]
Further to html, ok!, do you need me to review rest?

**Comment by Cartofante** [comment]
Frontend tests were OK :+1: ([details](http://clinker.cartodb.net/jenkins/job/CartoDB-PR-testing/4666/))

**Comment by juanignaciosl** [comment]
@xavijam no, that's enough, Ruby code is quite straightforward. Let's summon @rafatower just in case :)

**Comment by rafatower** [comment]
:+1:

File: spec/requests/signup_controller_spec.rb [hunk 1]
```diff
@@ -25,6 +25,16 @@
       response.status.should == 404
     end
 
+    it 'returns user error with admin mail if organization has not enough seats' do
+      fake_owner = FactoryGirl.build(:valid_user)
+      fake_organization = FactoryGirl.build(:organization, whitelisted_email_domains: ['cartodb.com'], seats: 0, owner: fake_owner)
+      Organization.stubs(:where).returns([fake_organization])
+      get signup_url
+      response.status.should == 200
+      response.body.should match(/Please, contact with the administrator of #{fake_organization.name}/)
```
Comment (rafatower) [inline spec/requests/signup_controller_spec.rb]: The problem with the spec seems to be that you don't have interpolation within that regexp. One quick solution:
```ruby
response.body.should match(/"Please, contact with the administrator of #{fake_organization.name}"/)
```
alternatively:
```ruby
response.body.should match(%r"Please, contact with the administrator of #{fake_organization.name}")
```

Comment (juanignaciosl) [inline spec/requests/signup_controller_spec.rb]: Thank you for taking a look at the test :-) Syntax was right (probably yours are as well), I had removed the "with" before "the administrator" from the final text and did not fix the regexp :sweat_smile:

**Comment by Cartofante** [comment]
Frontend tests were OK :+1: ([details](http://clinker.cartodb.net/jenkins/job/CartoDB-PR-testing/4673/))

**Pull request merged**